src/browser/ui/dom/components/ReactDOMTextarea.js

```diff
@@ -21,7 +21,6 @@ var ReactUpdates = require('ReactUpdates');
 
 var assign = require('Object.assign');
 var invariant = require('invariant');
-
```

Comment:
No, but I had removed `invariant` (opened wrong file for a different error) and then forgot to revert the space too. I'll take it out so it's not distracting.